pkg/sns/SnsProducer.php

```diff
@@ -51,7 +51,7 @@ public function send(Destination $destination, Message $message): void
             'MessageAttributes' => [
                 'Headers' => [
                     'DataType' => 'String',
-                    'StringValue' => json_encode([$message->getHeaders(), $message->getProperties()]),
+                    'StringValue' => base64_encode(json_encode([$message->getHeaders(), $message->getProperties()])),
```

Comment:
Maybe we can leave this without encoding base64 for now (keep changes not breaking). 
In the next major release, we can add encode part. 🤔